docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "What's new in accessibility in the .NET Framework"
+ms.custom: "updateeachrelease"
+ms.date: "10/13/2017"
+ms.prod: ".net-framework"
+ms.technology: 
+  - "dotnet-clr"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "what's new [.NET Framework]"
+author: "rpetrusha"
+ms.author: "ronpet"
+manager: "wpickett"
+---
+
+# What's new in accessibility in the .NET Framework
+
+The .NET Framework aims at making applications more accessibile for your users. Accessibility features allow an application to handle a range of devices and form factors seamlessly. Starting with the .NET Framework 4.7.1, the .NET Framework includes a large number of accessibility improvements that allow developers to create accessible applications. 
+
+The new accessibility features are enabled by default for applications that target the .NET Framework 4.7.1 or later. In addition, applications that target an earlier version of the .NET Framework but are running on the .NET Framework 4.7.1 or later can opt out of legacy accessibility behaviors by adding the following switch to the [`<AppContextSwitchOverrides>`](~/docs/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) element in the [`<runtime>`](~/docs/configure-apps/file-schema/runtime/index.md) section of the application's configuration file. 
```

Comment:
This isn't super clear that opting out of legacy behaviors is the same thing as opting in to the 4.7.1 improvements.